src/Microsoft.AspNet.Razor/CodeGenerators/CSharpTagHelperCodeRenderer.cs

```diff
@@ -688,6 +712,23 @@ private void RenderBufferedAttributeValueAccessor(CSharpCodeWriter writer)
             }
         }
 
+        private static bool ContainsChildContent(IList<Chunk> children)
+        {
+            foreach (var child in children)
```

Comment:
Ahh, I messed up line 725 :+1: . That should only return IF it's value is `true`. Thankfully today that edge case wont ever be hit due to how the parser parses but should correct the method nevertheless.